src/ui/components/VPNGetHelp.qml

```diff
@@ -32,9 +32,14 @@ Item {
         model: VPNHelpModel
 
         delegate: VPNExternalLinkListItem {
+            property var isMobileViewLogsLink: (window.fullscreenRequired() && id === 0)
```

Comment:
maybe we can add a boolean flag in the model to communicate when it's an external link.

Reply:
I can take this part.